pkg/apis/install/v1alpha1/doc.go

```diff
@@ -0,0 +1,4 @@
+// +k8s:deepcopy-gen=package
+// +groupName=install.karmada.io
```

Comment:
It is recommended to use `groupName=operator.karmada.io`

Reply:
if we have more crd, the groupName is hard to expand.
we can discuss it the next meeting.